src/components/app/MenuButtons/MetaInfo.js

```diff
@@ -87,6 +87,12 @@ export class MenuButtonsMetaInfo extends React.PureComponent<Props> {
                     )}
                   </div>
                 ) : null}
+                {meta.debug !== undefined ? (
+                  <div className="metaInfoRow">
+                    <span className="metaInfoLabel">Debug Build:</span>
+                    {meta.debug ? 'Yes' : 'No'}
+                  </div>
+                ) : null}
```

Comment:
The other option is here to make it `Build Type: Opt/Debug` instead of `Debug Build: Yes/No`. I couldn't decide :) What do you think?

Reply:
I think I prefer the first solution, I find it easier to read.